all/entrypoint.sh

```diff
@@ -19,7 +19,7 @@ printUsage() {
 }
 
 GEN_GATEWAY=false
-SUPPORTED_LANGUAGES=("go" "ruby" "csharp" "java" "python" "objc")
+SUPPORTED_LANGUAGES=("go" "ruby" "csharp" "java" "python" "objc" "node")
```

Comment:
What do you think about calling this `js` instead of `node`? I think it'd at least have consistency with the `$LANG_out`, and might be useful for other javascript frameworks.

Can you also update all/test.sh to include this in the list of languages to test? Line 3 in that file should be all that you have to change.

Reply:
The protoc pluging is called node, that's why I chose it. I think it's nodejs only as it stands. Can change it to js - then need to change the plugin part to not use the LANG but instead use a fixed js. Swings and roundabouts on that one.

I'll add the test.